tests/test_js_graph.py

```diff
@@ -660,6 +660,26 @@ def test_remove_cells(graph_cases) -> None:
     assert graph.get_edge(*vertices) is None
 
 
+def test_clone_cells(graph_cases) -> None:
+    """
+    :type graph_cases: qmxgraph.tests.conftest.GraphCaseFactory
+    """
+    graph = graph_cases('2v_1e')
+
+    vertices = graph.get_vertices()
```

Comment:
Not really a problem.
Do not need to change.

```suggestion
    vertices = graph.get_vertices()
    assert len(vertices) == 2  # sanity.
```

Reply:
I say @nicoddemus sugestion
https://github.com/ESSS/qmxgraph/pull/140#discussion_r1052320002
is nicer.